wdl2cwl/main_oop.py

```diff
@@ -24,32 +23,34 @@
 )
 
 
-class Converter:
-    """Object that handles WDL Workflows and task conversion to CWL."""
+def convert(doc: str) -> cwl.CommandLineTool:
```

Comment:
@mr-c this function was doing the conversion and handling serialization too. So I would have to pass the output parameter to choose between `stdout` or file output... I thought it simpler to make it do just the conversion (as per its own name) and leave the serialization up to the user. Now it returns a CWL object.